test/e2e-go/features/followerNode/syncDeltas_test.go

```diff
@@ -60,45 +58,26 @@ func TestBasicSyncMode(t *testing.T) {
 	err = fixture.ClientWaitForRoundWithTimeout(fixture.GetAlgodClientForController(nc), waitForRound)
 	a.NoError(err)
 
-	// Now spin up third node in follower mode
-	cloneDataDir := filepath.Join(fixture.PrimaryDataDir(), "../clone")
-	cloneLedger := false
-	err = fixture.NC.Clone(cloneDataDir, cloneLedger)
+	// Get the follower client, and exercise the sync/ledger functionality
+	followControl, err := fixture.GetNodeController("Follower")
```

Comment:
I suggest submitting some request that fails in follower mode to ensure the node is indeed is a follower